src/main/resources/db/migration/V001__CREATE_MISSIONS_TABLE.sql

```diff
@@ -0,0 +1,18 @@
+
+CREATE TABLE mission(
+  id text not null,
+  name text not null,
+  description text not null,
+  created_by text not null,
+  labels text[],
+  assignee_ids text[],
+  is_explored text not null,
+  status text,
+  sprint text,
+  team_id text not null,
+  critical text,
+  comments text[],
+  created_at timestamp without time zone,
+  updated_at timestamp without time zone,
+  primary key (id,team_id)
+);
```

Comment:
Do we need an updated_by also for audit?

Reply:
Audit had to be designed , but yes we need the updated_by